<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Dwipesh" ignored="0" name="Test" tests="2" failures="1" timestamp="23 Mar 2020 04:55:51 GMT" time="14.613" errors="0">
  <testcase name="test2" time="4.316" classname="com.d11.project.msd.masterTest.DemoTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.d11.project.msd.masterTest.DemoTest.test2(DemoTest.java:22)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase name="test" time="7.519" classname="com.d11.project.msd.masterTest.DemoTest"/>
</testsuite> <!-- Test -->
